src/main/groovy/com/mini/asaas/payment/JobScheduler.groovy

```diff
@@ -0,0 +1,29 @@
+package com.mini.asaas.payment
+
+import org.quartz.JobBuilder
+import org.quartz.SimpleScheduleBuilder
+import org.quartz.TriggerBuilder
+import org.quartz.impl.StdSchedulerFactory
+
+import java.util.concurrent.TimeUnit
+
+class JobScheduler {
+    static void startScheduler() {
+        def scheduler = StdSchedulerFactory.getDefaultScheduler()
+        scheduler.start()
+
+        def job = JobBuilder.newJob(ProcessOverduePayments)
+                .withIdentity("billingJob", "group1")
+                .build()
+
+        def trigger = TriggerBuilder.newTrigger()
+                .withIdentity("billingTrigger", "group1")
+                .startNow()
+                .withSchedule(SimpleScheduleBuilder.simpleSchedule()
+                        .withIntervalInSeconds(TimeUnit.DAYS.toSeconds(1) as Integer)
+                        .repeatForever())
+                .build()
+
+        scheduler.scheduleJob(job, trigger)
+    }
```

Comment:
Todo Job que for implementado precisará ser configurado de maneira explícita aqui?

Geralmente simplesmente criamos a classe de Job e configuramos a trigger direto nele

![image](https://github.com/EmilyFrade/mini-asaas/assets/62653479/b05dfc52-e8e6-44ba-88ae-debbd6fbe127)


Tentaram fazer algo assim?

Reply:
Nesse caso o controle do intervalo de execução fica onde? Definido através de cronExpression?

Reply:
Tentei não, vou alterar

Reply:
Isso, pela cron expression.

Dá pra fazer uma trigger simples também , com intervalo

![image](https://github.com/EmilyFrade/mini-asaas/assets/62653479/29d71627-e457-47c6-90b5-000e2e33ad4d)